rover_msgs/Obstacle.lcm

```diff
@@ -1,7 +1,6 @@
 package rover_msgs;
 
 struct Obstacle {
-	double bearing; // from straight ahead
-	double rightBearing;
-	double distance; // from straight ahead
-}
+	double bottom_left_coordinate_meters[3];
+	double top_right_corrdinate_meters[3];
+}
```

Comment:
wait doesn't this also need the ObstacleList.lcm from this [ICD](https://docs.google.com/document/d/14QWH8xS4Eq984rMYm_qVhqzoIqVrVtYsluBA5v8_nOQ/edit)?